docs/extensions-plugins/neovim/quickstart.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: Pieces Neovim Plugin Quickstart
+description: Learn how to set up and start using the Pieces Neovim Plugin.
+---
+
+import CTAButton from "/src/components/CTAButton";
+import { pieces_app } from "/src/lib/const";
+
+# Quickstart
+
+In this tutorial, we'll walk you through the process of setting up and using the Pieces Neovim Plugin. We'll cover the installation process and basic usage to help you get up and running quickly.
+
+<CTAButton
+  href={pieces_app.links.extensions.neovim}
+  label={'Install via GitHub'}
+  icon={'/assets/pieces_logos/white_circle.png'} 
+  iconDark={'/assets/pieces_logos/black_circle.png'} 
+  type={'secondary'}
+/>
+
+## Prerequisites
+
+- [Pieces OS](/installation-getting-started/what-am-i-installing) installed on your machine.
+- [Neovim](https://neovim.io/) installed on your machine.
+- [Python](https://www.python.org/downloads/) installed on your machine because Python is required to run Python based Neovim plugins.
+
+A few things to note before you start installing the Pieces Neovim Plugin:
+
+- The Python based Neovim plugins requires the `pynvim` Python package, which is not included in the default Neovim installation. You can install it by running the following command:
+
+```bash
```

Comment:
Should be indented

Reply:
@shivay-at-pieces still isn't indented correctly, you just need move the code block over 4 spaces.

Same goes for the others below